File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v1/MixAccountApi.java
package com.bitget.openapi.api.v1;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface MixAccountApi {

    @GET("/api/mix/v1/account/account")
    Call<ResponseResult> account(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/account/accounts")
    Call<ResponseResult> accounts(@QueryMap Map<String, String> paramMap);

    @POST("/api/mix/v1/account/setLeverage")
    Call<ResponseResult> setLeverage(@Body Map<String, String> paramMap);

    @POST("/api/mix/v1/account/setMargin")
    Call<ResponseResult> setMargin(@Body Map<String, String> paramMap);

    @POST("/api/mix/v1/account/setMarginMode")
    Call<ResponseResult> setMarginMode(@Body Map<String, String> paramMap);

    @POST("/api/mix/v1/account/setPositionMode")
    Call<ResponseResult> setPositionMode(@Body Map<String, String> paramMap);


    // position
    @GET("/api/mix/v1/position/singlePosition")
    Call<ResponseResult> singlePosition(@QueryMap Map<String, String> paramMap);

    @GET("/api/mix/v1/position/allPosition")
    Call<ResponseResult> allPosition(@QueryMap Map<String, String> paramMap);
}

File: bitget-php-sdk-api/src/api/v1/MixAccountApi.php
<?php


namespace bitget\api\v1;


use bitget\internal\BitgetApiClient;

class MixAccountApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function account($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/account/account", $params);
    }

    public function accounts($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/account/accounts", $params);
    }

    public function setLeverage($params): string
    {
        return $this->BitgetApiClient->doPost("/api/mix/v1/account/setLeverage", $params);
    }

    public function setMargin($params): string
    {
        return $this->BitgetApiClient->doPost("/api/mix/v1/account/setMargin", $params);
    }

    public function setMarginMode($params): string
    {
        return $this->BitgetApiClient->doPost("/api/mix/v1/account/setMarginMode", $params);
    }

    public function setPositionMode($params): string
    {
        return $this->BitgetApiClient->doPost("/api/mix/v1/account/setPositionMode", $params);
    }


    // position
    public function singlePosition($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/position/singlePosition", $params);
    }

    public function allPosition($params): string
    {
        return $this->BitgetApiClient->doGet("/api/mix/v1/position/allPosition", $params);
    }
}